redirect/routes.php

```diff
@@ -71,10 +71,14 @@ class Route {
         'anal'         => [
             'description' => 'Analysis für Informatiker',
             'target'      => 'https://www-m5.ma.tum.de/Allgemeines/MA0902_2017W',
+             // If you change this moodle_id please also change in the MainTest.php, 
+             // since this url is also used for testing!
```

Comment:
The test will fail in any case, no added benefit to this comment :man_shrugging:

Reply:
Why will it fail?